packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -0,0 +1,74 @@
+// @ts-check
+
+import { makePromiseKit } from '@agoric/promise-kit';
+import { makeNotifierKit } from '@agoric/notifier';
+import { assert } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+
+import '../types';
+import '../internal-types';
+
+/** @type {MakeZoeSeatAdminKit} */
+export const makeZoeSeatAdminKit = (
+  initialAllocation,
+  instanceAdmin,
+  proposal,
+  brandToPurse,
+  promises = {},
+) => {
+  const payoutPromiseKit = makePromiseKit();
+  const offerResultPromiseKit = promises.offerResult || makePromiseKit();
+  const exitObjPromiseKit = promises.exitObj || makePromiseKit();
```

Comment:
There is no reason for these named parameters to be promiseKits. `makeZoeSeatAdminKit` only ever makes of the promises of those kits. Only the caller ever makes use of the resolvers.